### PR TITLE
added follower/following list

### PR DESCRIPTION
**Pull request by JenniceLee0318**

### CODE REVIEW

**Review by sanjayalwani — COMMENTED** [review]
This is a start, we'll need to include the follower/following list in the user page, note that we've also queried the friends table in our get user profile request so we have that available on initialization.
Also I think the follower following naming for user profile is the other way around, people who are following the user are the "followers" and people who are being followed by the user are the people the user is "following".

File: index.js [hunk 1]
```diff
@@ -119,8 +119,7 @@ app.post("/reguser", (req, res) => {
   let check = false;
   let inputplaced;
   pool.query(
-    `SELECT * FROM users WHERE login = $1`,
-    [username],
+    `SELECT * FROM users WHERE login='${username}'`,
```
Comment (mann-c) [inline index.js:129]: why did you change this, parameters should be used

File: index.js [hunk 2]
```diff
@@ -140,8 +139,7 @@ app.post("/reguser", (req, res) => {
         });
       } else {
         pool.query(
-          `INSERT INTO users VALUES($1,$2,$3,$4,'',$5)`, //Empty string between $4, $5 is the empty description
-          [username, first_name, last_name, city, password],
+          `INSERT INTO users VALUES('${username}','${first_name}','${last_name}','${city}','','${password}')`,
```
Comment (mann-c) [inline index.js:149]: parameters should be used

File: index.js [hunk 3]
```diff
@@ -196,24 +194,16 @@ app.get('/restaurant/:uid', checkNotAuthenticated, (req, res) => {
   var uid = req.params.uid;
   console.log(uid);
 
-  var query = `select * from restaurants where id=$1`;
+  var query = `select * from restaurants where id=${uid}`;
```
Comment (mann-c) [inline index.js:204]: should be using parameters

**Review by mann-c — CHANGES_REQUESTED** [review]
why did you undo all of the changes to index.js that sanjay made on thursday?

**Pull request closed without merging**